solr/core/src/java/org/apache/solr/schema/IndexSchema.java

```diff
@@ -1981,4 +2089,10 @@ public PayloadDecoder getPayloadDecoder(String field) {
     return decoders.computeIfAbsent(ft, f -> PayloadUtils.getPayloadDecoder(ft));
   }
 
+  @Override
+  public void close() throws IOException {
+    if (pluginLoader instanceof PackageAwarePluginLoader) {
```

Comment:
Why the instanceof check here?

Reply:
defensive , programming 😉

Reply:
Ok, I see - we create this ourselves and then we have to close it :)